src/includes/content/privacy_policy/simple.pug

```diff
@@ -147,3 +147,6 @@
         | Markdown
       button.button.is-success(v-if='contentRenderType!=1' @click='preview()')
         | Preview
+      div(style='flex:1')
+        button.button.is-info(@click="fc_deploy()" style='float:right;')
+          | Deploy
```

Comment:
We shouldn't need it here, because there is a proper block for this in flycricket.pug

Reply:
One more thing. I wanted to put flycricket behind a flag, since it is an integration and I should be able to disable it completely when I want to.

So check here how I put it behind a flag: https://github.com/nisrulz/app-privacy-policy-generator/blob/develop/src/includes/content/wizard/step3.pug#L30

I would opt for putting this button behind the flag. Same for other one. 

This is just a pattern I want to follow for future updates also. These integrations should be easy enough to toggle by someone.

Reply:
Got it - just pushed a new commit with these buttons flagged for both privacy and tnc. Ran a few tests compiling with the flag on and off, and looks to be working well from my end! 

Also updated the server so the deployment is backward-compatible and can now be rolled out at any time.